UI/views/courses.html

```diff
@@ -68,7 +68,9 @@ <h3 class="colHeader"> <span class="sr-only">{{lang.panelTitle}}</span>
               <div class="col-lg-4 col-md-4 col-sm-4 col-xs-4" style="padding-right:0;text-align:right"><small>({{Meeting.Location}})</small></div>
             </div>
             <div class="row" ng-show="filteredDays.length === 0">
-              <div ng-hide="showAllMeetings" class="col-lg-12 col-md-12 col-sm-12 col-xs-12" style="padding-left:0"><small><a href="" ng-click="showAllMeetings = true">Between terms. Show all meetings?</a></small></div>
+              <div ng-hide="showAllMeetings" class="col-lg-12 col-md-12 col-sm-12 col-xs-12" style="padding-left:0"><small><a href="" ng-click="showAllMeetings = true">No meetings now. Show all meetings?</a></small></div>
```

Comment:
This message is unclear to me.  Can 'all' be scoped so it doesn't sound like all meetings over all time?  E.g. 'Show meetings before / after this term.' or 'No current meetings.  Show all meetings for class?'